addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -4,7 +4,7 @@
 <template id="courses_home_side_bar">
     <div class="container o_wslides_home_main">
         <div class="row">
-            <t t-set="has_side_column" t-value="is_view_active('website_slides.toggle_leaderboard')"/>
+            <t t-set="has_side_column" t-value="not is_public_user"/>
             <t t-if="is_public_user">
                 <div t-if="has_side_column">
```

Comment:
We should add a comment to clean this in master as this will never be shown (and remove the row div).